### PR TITLE
Feature : Add delete file feature.

### LINKED ISSUE #478: Add delete file feature in Data Logger Screen.
**Expected Behaviour**  
A delete button should to added beside rename button to delete files.
As of now, we have rename feature there and no delete option. 

**Additional context**  
After adding delete button : -
Screenshot from 2019-09-24 12-19-15

**Would you like to work on the issue?**
Yes.

### PR DESCRIPTION
**Pull request by dipu989**
Fixes #478 

**Changes**:
1) Added delete file feature.

**Screenshot/s for the changes**:
deleteGif

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 

[apkDebug.zip](https://github.com/fossasia/neurolab-android/files/3657394/apkDebug.zip)

### CODE REVIEW

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Nice work :+1: please use resources instead of hard coded values. Rest looks good.

**Comment by dipu989** [comment]
@CloudyPadmal Removed hard-coded string. Please check!

File: app/src/main/res/layout/logged_data_item.xml [hunk 1]
```diff
@@ -47,10 +47,11 @@
             android:id="@+id/rename_file_btn"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginEnd="@dimen/layout_margin_medium"
+            android:layout_marginEnd="8dp"
```
Comment (jddeep) [inline app/src/main/res/layout/logged_data_item.xml]: Use dimens resources.

File: app/src/main/res/layout/logged_data_item.xml [hunk 2]
```diff
@@ -73,6 +74,18 @@
             app:layout_constraintEnd_toStartOf="@+id/focus_play"
             app:layout_constraintTop_toTopOf="parent"
             app:srcCompat="@drawable/ic_card_mem_graph" />
+
+        <ImageView
+            android:id="@+id/delete_file_btn"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="8dp"
```
Comment (jddeep) [inline app/src/main/res/layout/logged_data_item.xml]: Remove hard coding.

**Review by jddeep — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/logged_data_item.xml (hunk 2 above)
Comment (dipu989) [reply app/src/main/res/layout/logged_data_item.xml]: @jddeep Done with the changes! Please check.

**Review by jddeep — APPROVED** [review]

**Pull request merged**